### PR TITLE
Fix layout related issues #737, #667, #704

### PR DESCRIPTION
**Pull request by nvasilas**
Use tmux default session size 80x24 when creating a new session

re: #737, #667, #704

### CODE REVIEW

**Comment by tony** [comment]
@nvasilas Can you take a look at the tests?

It looks like this may need a version conditional from libtmux.common, example:

- https://github.com/tmux-python/tmuxp/blob/ad6d30b7c0ed26df1160deb53092b913ae690651/tmuxp/cli/load.py#L165
- https://github.com/tmux-python/tmuxp/blob/9a627f6949c945ad3427b3c559e0705042d0b0e8/tests/test_workspacebuilder.py#L255

**Comment by tony** [comment]
Docs on related functions: https://libtmux.git-pull.com/reference/common.html?highlight=has_gt_version#libtmux.common.has_gt_version

**Comment by nvasilas** [comment]
@tony Sure mate I 'll fix them.

**Comment by tony** [comment]
@nvasilas I rebased your branch via GitHub's rebase button (`git pull --rebase --autostash` to pull the changes)

**Comment by tony** [comment]
@nvasilas It will works O_O

Tested by doing `pip install -e ~/path/to/your/tmuxp/branch` and doing `tmuxp load ./workspacefile.yaml -L another-tmux-server`.

Master: b70cd70: Layout is scrunched
This PR: Maintains the layout!

**Comment by tony** [comment]
@nvasilas Thank you for this!

Live in v1.13.1 

[PyPI](https://pypi.org/project/tmuxp/1.13.1/), [GitHub Tag](https://github.com/tmux-python/tmuxp/tree/v1.13.1)

Any better?

I will reach out to users in the other issues as well

File: tmuxp/workspacebuilder.py [hunk 1]
```diff
@@ -264,9 +269,6 @@ def build(self, session=None, append=False):
                 assert isinstance(p, Pane)
                 p = p
 
-                if "layout" in wconf:
-                    w.select_layout(wconf["layout"])
-
```
Comment (tony) [inline tmuxp/workspacebuilder.py:269]: @nvasilas Do you recall the rationale behind the removing / adding of the `.select_layout`'s here, [here](https://github.com/tmux-python/tmuxp/pull/793/files#diff-9316655a97cbe443559ee45c1b1fa26bfa86e314210653806ce215702a956931R286), and [here](https://github.com/tmux-python/tmuxp/pull/793/files#diff-9316655a97cbe443559ee45c1b1fa26bfa86e314210653806ce215702a956931L424)?

Comment (nvasilas) [reply tmuxp/workspacebuilder.py:269]: @tony If I recall correctly without this change the layout wasn't updated properly. It seems like I broke some things with this PR. I will investigate and I'll try to provide a fix.

Comment (tony) [reply tmuxp/workspacebuilder.py:269]: @nvasilas Thank you! Let's see how it goes!

File: tests/test_workspacebuilder.py [hunk 2]
```diff
@@ -1183,3 +1183,37 @@ def f():
...
+    assert len(window.panes) == 3
+    main_horizontal_pane, *panes = window.panes
+
+    def height(p):
+        return int(p._info["pane_height"])
+
+    def width(p):
+        return int(p._info["pane_width"])
+
+    assert height(main_horizontal_pane) > height(panes[0])
```
Comment (tony) [inline tests/test_workspacebuilder.py:1212]: I learned something new:

`main-horizontal` doesn't necessarily guarantee the top pane will be bigger.  

> A large (main) pane is shown at the top of the window and the remaining panes are spread from left to right in the leftover space at the bottom. Use the main-pane-height window option to specify the height of the top pane. [manpage](https://manpages.debian.org/buster/tmux/tmux.1.en.html#main-horizontal)

I will try to remedy this in #809

**Comment by tony** [comment]
@nvasilas I am deliberating fully reversing the change while we understand the problem better. You can recreate the same 

The reason why I may reverse it (v1.13.2 and v1.13.3) is it had more of an effect on users with certain kinds of sessions than we both thought it would (#800)

If you have more ideas on this - I am all ears - a bulk of my effort is improving `libtmux` so issues like this are easier to diagnose. The experience is going to get better, and we'll improve at catching these issues, I promise.

In #809 I added a test for us to check for tmux spacing issues

**Comment by tony** [comment]
@nvasilas I've started a new issue at #815 the opportunity arises you'd like to collab (sorry about retracting on this PR, it is my 100% my mess up)

**Comment by tony** [comment]
@nvasilas I will keep you apprised either way 👍 (and vice-versa, hopefully!)

**Pull request merged**